src/components/Toolbelt/Toolbelt.test.js

```diff
@@ -1,39 +1,61 @@
 import React from 'react'
-import { shallow } from 'enzyme'
+import { render, screen } from '@testing-library/react'
+import userEvent from '@testing-library/user-event'
 
 import { fieldMode } from '../../enums'
 
 import { Toolbelt } from './Toolbelt'
 
-let component
-
-beforeEach(() => {
-  component = shallow(
-    <Toolbelt
-      {...{
-        fieldMode: fieldMode.OBSERVE,
-        handleFieldModeSelect: () => {},
-      }}
-    />
-  )
-})
-
-test('renders shop inventory', () => {
-  expect(component).toHaveLength(1)
-})
-
-describe('tool selection', () => {
-  test('selects no tools by default', () => {
-    expect(component.find('.selected')).toHaveLength(0)
+let buttons = []
+
+class ToolbeltTest extends React.Component {
```

Comment:
this is not going to be the response you were expecting i don't think... but i actually did this just because i wanted to be able to test clicking the button and having it become selected. you were using the noop for the click callback and this was the only way i could think to really hook it up, but now that you called it out i thought some more and the unit test here is simply that it marks the expected button selected based on the fieldMode prop. i added new tests to do that, i also added a "visually hidden" class to put some text in there that isn't actually seen. this should be helpful to screen readers as well, though we have a lot more work to do in that regard.

Reply:
The new approach is WAY nicer, thank you for taking another stab at this! I think it's simple, effective, and scalable. Nice work!

Reply:
🙌